components/director/internal/domain/application/service.go

```diff
@@ -790,6 +791,29 @@ func (s *service) GetLabel(ctx context.Context, applicationID string, key string
 	return label, nil
 }
 
+// GetScenariosLabelGlobal missing godoc
+func (s *service) GetScenariosLabelGlobal(ctx context.Context, applicationID string) (*model.Label, error) {
```

Comment:
If the application is in different formations in different tenants, it will have multiple scenario labels, therefore this probably needs to be List + merge all values in the application code.